client/src/reducers/user.js

```diff
@@ -0,0 +1,73 @@
+import * as Actions from "../actions/actionDefinitions";
```

Comment:
I like the idea of a user reducer/store slice. I'm wondering if we should roll `auth` into it...